PhysicsTools/ONNXRuntime/src/ONNXRuntime.cc

```diff
@@ -0,0 +1,161 @@
+/*
+ * ONNXRuntime.cc
+ *
+ *  Created on: Jun 28, 2019
+ *      Author: hqu
+ */
+
+#include "../interface/ONNXRuntime.h"
+
+#include <cassert>
+#include <iostream>
+#include <algorithm>
+#include "FWCore/Utilities/interface/Exception.h"
+
+namespace Ort {
+
+  const OrtApi* g_api = OrtGetApi(ORT_API_VERSION);
```

Comment:
What is the purpose of this global variable that looks to be unused?

Reply:
It's needed because of https://github.com/microsoft/onnxruntime/blob/master/include/onnxruntime/core/session/onnxruntime_cxx_api.h#L28.

Reply:
Could you add a comment to briefly explain that?